src/vs/platform/quickinput/common/quickInput.ts

```diff
@@ -359,6 +364,11 @@ export interface IQuickInput extends IDisposable {
 	 */
 	ignoreFocusOut: boolean;
 
+	/**
+	 * An optional anchor for the quick input.
+	 */
+	anchor?: HTMLElement | { x: number; y: number };
```

Comment:
This being supported means that showing could happen first:
```ts
qp.show();
qp.anchor = foo;
```

we need to update the anchor when it is set.